[RU] Add `Beatmap nominator test`, `Beatmap veto`, and `Becoming a beatmap nominator` (#7943)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/ru.md

# Как стать номинатором

За зачисление в [номинаторы](/wiki/People/The_Team/Beatmap_Nominators) отвечает [команда оценки номинаций](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT).

Заявку на вступление в команду номинаторов нужно заполнять на [веб-сайте для NAT](http://bn.mappersguild.com/bnapps).

Для подачи заявки необходимо подходить под требования по моддингу. В форме нужно указать ссылки минимум на две карты, которые вы недавно модили, а также пройти [тест на номинатора](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Nominator_Test), составленный по [критериям ранкинга](/wiki/Ranking_Criteria). NAT свяжется с вами после рассмотрения заявки. Остальные подробности о подаче заявки см. на веб-сайте для NAT.

## Базовые требования

От моддеров, желающих стать номинаторами, ожидается следующее:

- Активный моддинг в течение минимум 3 месяцев перед подачей заявки;
- Как минимум 150 или 200 набранных [кудосу](/wiki/Modding/Kudosu) (зависит от режима игры);
- Ссылка на 2–3 мода, демонстрирующих навыки кандидата в роли номинатора.

### Качество моддинга

Моды, приводимые в качестве примеров, должны отражать навык анализа и улучшения карт на уровне номинатора.

> Ниже перечислены качества, которыми должны обладать номинаторы. Вашу заявку могут отклонить, если в перечисленных в ней модах не видно хотя бы одной из этих черт.

Reviewer: ```suggestion
Ниже перечислены качества, которыми должны обладать номинаторы. Вашу заявку могут отклонить, если в перечисленных в ней модах не будет видно какую-либо из этих черт.
```
В оригинале any of these, а не some of these, то есть скорее имеется в виду, что если чего-то из этого списка нет в примерах заявки, то её могут отклонить. Не будут же брать номинаторов, которые умеют только замечать неранкабельные проблемы, а всё остальное не умеют.

Author: так и есть, и моя формулировка это и описывает: если хоть один пункт не проявлен, заявку могут (пусть и формально) отклонить

- **Моддинг всех уровней сложности;**
- **Умение замечать неранкабельные проблемы, включая те, которые нельзя обнаружить автоматически (например, неправильный тайминг или метаданные);**

> - **Сопоставление разных участков карты как аргументация своих замечаний;**

Reviewer: ```suggestion
- **Сравнение разных участков карты для поддержки аргументации своих замечаний;**
```
Замудрёно слишком написано.

Author: да, у тебя лучше

- **Замечания, направленные на разные стороны карты, например, на ритмику, спейсинг, движение, интенсивность, контраст и последовательность;**

> - **Умение находить как конкретные недочёты, так и проблемы, присущие всей карте целиком;**

Reviewer: ```suggestion
- **Умение находить как конкретные недочёты, так и проблемы, присущие всей карте;**
```
Вся карта и так подразумевает целиком.

Author: я хотел сказать «в целом», систематические ошибки, иным словом

> - **Понимание намерений маппера при поиске проблем или возможных улучшений.**

Reviewer: ```suggestion
- **Понимание задумки маппера при поиске проблем или возможных улучшений.**
```

### Процесс отбора заявкок

- Поданные заявки проверяют члены NAT соответствующего игрового режима. Они обращают внимание на следующие стороны:

>   - Общее поведение: тон общения и подача модов, а также другие пункты из [соответствующего свода правил](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping), включая поведение в чате и на форумах в принципе.

Reviewer 2: ```suggestion
  - Общее поведение: тон общения и подача модов, а также другие пункты из [кодекса поведения](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping), включая поведение в чате и на форумах в принципе.
```

Author: слишком много `поведений` на квадратный сантиметр экрана будет

>   - Навыки моддинга: знание [критериев ранкинга](/wiki/Ranking_Criteria) — как общих, так и для отдельных режимов. Также учитываются дополнительные умения (поиск метаданных, работа с таймингом и т.д.)

Reviewer 3: ```suggestion
  - Навыки моддинга: знание [критериев ранкинга](/wiki/Ranking_Criteria) — как общих, так и для отдельных режимов. Также учитываются дополнительные умения (поиск метаданных, работа с таймингом и т.д.).
```

- После того, как все заявки рассмотрены, члены NAT обсуждают кандидатов и решают, кто из них готов стать номинатором.

> ## Уровень моддинга

Reviewer 2: ```suggestion
## Уровень активности
```

Reviewer 2: Или
Активность моддера

Author: допустим

Для того, чтобы стать номинатором в режиме osu!, необходимо иметь определённый уровень активности. Это требование пока **не** применяется к прочим режимам (osu!taiko, osu!catch и osu!mania).

> ### Какие моды учитываются

Reviewer 2: Это точно нужно переписать более понятным образом
Но я пока не придумал
"Критерии деятельности мода", наверное

Author: мод здесь — это отдельный пост на сайте, «деятельность мода» звучит примерно как «деятельность сообщения» (так же бессвязно). придумаю свой вариант

- После авторизации на веб-сайте для NAT вы увидите число кудосу, полученных за последние 3 месяца за мод-посты.
- При подсчёте кудосу учитываются только посты за последние 3 месяца в обсуждениях карт, в которых хотя бы одно предложение или проблема были одобрены.
- Режим карты, которую вы модили, не учитывается, поскольку измеряется ваша активность в моддинг-сообществе в целом.
- Дата выдачи кудосу **имеет значение**, так как при подсчёте используется ваша история моддинга из профиля.

### Требования по баллам

- Требования зависят от того, во сколько баллов — `ModScore` — оценивается заявка:

![Формула ModScore](/wiki/shared/Modscore_new_wiki.png "Формула ModScore")

> `m` — сколько минимум различных карт необходимо моддить в месяц;\

Reviewer 3: ```suggestion
`m` — сколько минимум различных карт необходимо моддить в месяц;
```

Author: it's a line break required for two lines to stack -- see https://osu.ppy.sh/wiki/en/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator#how-the-required-minimum-score-is-calculated

`M` — на сколько различных карт вы написали моды за месяц.

- На главной странице [сайта для подачи заявок](http://bn.mappersguild.com/bnapps) можно увидеть, сколько `ModScore` вы набрали.
- Для учёта заявки необходимо, чтобы сумма `ModScore` за последние три месяца была неотрицательной.
- Требования и параметры формулы:
  - osu!taiko, osu!catch и osu!mania: `m` = 3, число кудосу — от 150
  - osu!: `m` = 4, число кудосу — от 200
- Старайтесь набирать чуть больше баллов, чем необходимо, поскольку требования могут слегка меняться в зависимости от числа заявок.

> ## Кулдаун

Reviewer 2: "Пауза перед новой заявкой (англ. Cooldown; жарг. Кулдаун)"?

Author: я к этому моменту уже сдался, тем более что где-то в help centre решили оставить так за неимением нормального аналога. подумаю ещё

| :-- | :-- | :-- |
| По умолчанию | 90 дней | `ModScore` больше нуля за последние 3 месяца |
| Сниженная длительность | 60 дней | `ModScore` больше нуля за последние 3 месяца |
| Проваленный [тест на номинатора](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Nominator_Test) | 30 дней | `ModScore` больше 0 за последние 3 месяца |

### После исключения из номинаторов

При исключении из номинаторов требования к повторной заявке могут отличаться от первоначальных. Номинаторы, хорошо справлявшиеся с работой, могут вступить в команду на упрощённых условиях, о чём при необходимости им сообщит член NAT.

Если вы уже были номинатором и после своего ухода не проваливали соответствующий тест, ориентируйтесь на требования в табличке внизу.

Требования к `ModScore` действуют только на режим osu!. Если вас исключили из-за нехватки активности, необходимые 8 модов можно набрать в любом режиме игры.

| Тип кулдауна | Длительность | Требования |
| :-- | :-- | :-- |
| По умолчанию | 60 дней | `ModScore` больше 0 за последние 60 дней |
| Неактивность | 30 дней | Любое значение `ModScore`, 8 модов за последние 60 дней |
| Хорошая активность | 30 дней | `ModScore` больше 0 за последние 30 дней |

> Полноправные номинаторы, к которым не было претензий, могут в течение года вступить обратно сразу же и без учёта каких-либо требований, если они ушли по собственному желанию.

Reviewer: ```suggestion
Полноправные номинаторы, могут в течение года вступить обратно сразу же и без учёта каких-либо требований, если они ушли по собственному желанию.
```
По собственному желанию и к которым не было претензий — одно и то же? Какую-то из этих частей можно убрать, чтобы не было повторений.

Author: не совсем: можно уйти как самостоятельно, а можно добиться того, чтобы тебя исключили, при этом первый случай не исключает наличия проблем: например, человек ещё не «довёл дело» до момента, когда его хотят выкинуть, но уже решил, что номинаторство ему не подходит
